circleci/templates/maven/config_template-no-docker-v1.yml

```diff
@@ -109,13 +116,47 @@ jobs:
       - image: circleci/ruby:2.4.1
     steps:
       - run: echo "*PLACEHOLDER* pretending to execute integration testing phase!"
-  Deploy Artifacts: # Deploy artifacts to Nexus or Maven Central
-    docker:
-      - image: circleci/ruby:2.4.1
-    steps:
-      - run: echo "*PLACEHOLDER* pretending to deploy build artifacts to Maven Central!"
 # END: UNDER CONSTRUCTION AREA ---------------------------------------
 
+  Tag Release:
+    docker: *image
+    steps:
+      # Tag Release creates a Git tag with given a given name and pushes it
```

Comment:
can we change this to "with given name" instead of the 2 givens?

Reply:
Yeah, stupid typo